src/styles/bloks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  @include flexProperties ($justifyContent: none);
+
+  box-sizing: border-box;
```

Comment:
Move the box-sizing to the * selector to ensure a consistent box model throughout your page